src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -350,3 +350,20 @@
   {% endif %}
   </div>
 {% endmacro %}
+
+{% macro multistore_config_row(isMultishopConfigEnabled, form, options = {}) %}
```

Comment:
That's nice but I think you can do even better by integrating this directly in the prestashop form theme, I think it can be added here:
https://github.com/PrestaShop/PrestaShop/blob/83a93eb14d6797144bd941d41dde0b7cd3ac6c17/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig#L54

This way you will only have to use a single `form_widget(form)` in your twig template and it will simplify the integration for other configuration forms a lot

You might need to synchronize with @JevgenijVisockij since he is already simplifying them this way, just to make sure we don't have two PRs doing the same thing

Reply:
Indeed it would be great to integrate this with form theme in some way. Maybe we can integrate it into form_widget and pass a parameter as part of parameter(or change default type to have such parameter)?

Reply:
I thought about form_row because the checkbox needs to be integrated in the same row as the input it is associated to, into an additional column
I'm not sure I get what you mean by the parameter, I thought the idea of the form theme was to handle as much as possible things automatically

I thought maybe the extension can add into the initial form field some `vars` containing the additional multistore checkbox so that in the theme we could do something like
```twig
{% if form.vars.multistore_check is defined %}
    {{ form_widget(form.vars.multisotre_check) }}
{% endif %}
```

Reply:
Well I mentioned the parameter because the form needs to know somehow, is this field multistore or not, right?
If we can get that info automatically then yeah I think your solution works.

Reply:
done !